admin/tools/docker/mariadb/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM mariadb:10.1.25
+MAINTAINER Fabrice Jammes <fabrice.jammes@in2p3.fr>
+
+# Start with this long step not to re-run it on
+# each Dockerfile update
+# TODO manage scisql dependencies with eups
+RUN apt-get -y update && \
+    apt-get -y upgrade && \
+    apt-get -y install apt-utils && \
+    apt-get -y install gcc git make libmariadbclient-dev \
+                       python python-dev python-pip && \
+    apt-get -y clean
+
+RUN apt-get -y install libssl-dev openssl
+
+# TODO manage scisql dependencies with eups
+RUN pip install MySQL-python future
```

Comment:
Are all these dependencies needed to just build scisql?

Reply:
yes, will switch to use in a future ticket.